distribution/rules.bzl

```diff
@@ -86,57 +78,152 @@ def _distribution_impl(ctx):
 
     return DefaultInfo(data_runfiles = ctx.runfiles(files=[ctx.outputs.distribution]))
 
+def _tgz2zip_impl(ctx):
+    ctx.actions.run(
+        inputs = [ctx.file.tgz],
+        outputs = [ctx.outputs.zip],
+        executable = ctx.file._tgz2zip_py,
+        arguments = [ctx.file.tgz.path, ctx.outputs.zip.path],
+        progress_message = "Converting {} to {}".format(ctx.file.tgz.short_path, ctx.outputs.zip.short_path)
+    )
+
+    return DefaultInfo(data_runfiles = ctx.runfiles(files=[ctx.outputs.zip]))
+
 
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
 def deploy_deb(name,
-               package_dir,
-               maintainer,
-               version_file,
-               description,
-               postinst = None,
-               prerm = None,
-               target = None,
-               empty_dirs = [],
-               files = {},
-               depends = [],
-               symlinks = {},
-               modes = {}):
+                package_name,
+                installation_dir,
+                maintainer,
+                version_file,
+                description,
+                target = None,
+                empty_dirs = [],
+                files = {},
+                depends = [],
+                symlinks = {},
+                permissions = {}):
     java_deps_tar = []
     if target:
+        java_deps_name = "_{}-deb-deps".format(package_name)
         java_deps(
-            name = "_{}-deps".format(name),
-            targets = [target],
+            name = java_deps_name,
+            target = target,
             java_deps_root = "services/lib/"
```

Comment:
what is `java_deps_root` and why is it hard-coded to `services/lib/`

Reply:
@lolski
>what is java_deps_root

folder inside the `tar` archive to put JARs into

>why is it hard-coded to services/lib/

that's the sensible default for `grakn-core-server`/`grakn-core-console`

Reply:
makes sense